serverless.yml

```diff
@@ -29,6 +29,13 @@ functions:
       - http:
           path: /{url}
           method: get
+  siebel-api:
+    handler: api/siebel.handler
+    timeout: 300
```

Comment:
Time can probably be less than 5 minutes if this is just handling a single api call. Probably 10 - 30 seconds.